### DISCUSSION TITLE
Portuguese localizations shouldn't translate variables

### DISCUSSION THREAD

**Discussion by ericnewcomer** [nyaruka/floweditor#1136] [Ideas]
The coded variables like `[[field]]` should not be translated, ie `[[campo]]`. They are place holder variable names that should be in english always.

https://github.com/nyaruka/floweditor/blob/08a322f53228a2be291765c35a6aa672b331de72/src/config/i18n/pt/resource.json#L287

**Comment by ericnewcomer** [comment]
@johncordeiro

**Comment by johncordeiro** [comment]
hey @ericnewcomer we're updating Transifex with the right variable names in the translation.
